src/urbit/mar/chess/moves.hoon

```diff
@@ -0,0 +1,28 @@
+/-  chess
+=,  format
+|_  moves=wain
+++  grab
+  |%
+  ++  noun  wain
+  --
+::
+::  going to frontend en:js form
+++  grow
+  |%
+  ++  noun  moves
+  ++  json
+    %-  frond:enjs
+    :-  'moves'
+    :: an array
+    :-  %a
+    :: turn the different lines of a
+    :: wain into an entry of %s string
+    :: for the array.
+    ::
+    ::  turn([list gate])
+    %+  turn  moves
+    :: will I need (of-wain move)
+    |=(move=@t [%s move])
+  --
+++  grad  %txt
+--
```

Comment:
See comment on `/mar/chess/game.hoon`.

Reply:
oops. this file doesn't need to be here. that was when I try to make `moves` a first-class thing. I didn't realize we were already pulling `moves` from a `GameInfo` type. 

I'll delete this file.